.storybook/components/SearchStory.js

```diff
@@ -14,15 +14,16 @@ storiesOf('Search', module)
       without the use of navigation. Search can be used as the primary means of discovering content,
       or as a filter to aid the user in finding content.
     `,
-    () => (
+    () =>
       <Search
+        key="normal"
         {...searchProps}
         className="some-class"
         id="search-1"
         labelText="Search"
```

Comment:
Sorry, unrelated to the PR but related to Search:

I think we need two variants of Search controlled by props.

As I'm using the components more, I'm finding the need to have access to either an `input` or `label + input` together. I mentioned to @iangfleming on his [Slider PR](https://github.com/carbon-design-system/carbon-components-react/pull/65#discussion_r126332560) that we should start offering 2 variants of our form elements controlled by props so that the user can choose which variant they need.
Also, It looks like we never actually show the `<label>`for it to be usable. This could be confusing for users who may try to use their own `<label>` with `Search` component. If we decide